magento/magento2
# Fix web setup - loosing admin path value and fix checking loading messages

### Description (*)
This PR fix the loosing admin path value when navigate the steps forward them backward.
Also fix some messages that was not displaying properly to user while performing some verifications when click on next button on step 2, 4 and 5.

### Fixed Issues (if relevant)
1. magento/magento2#27592: Web install - step 3- web configuration urls are always reset to default
2. magento/magento2#27587: Web install - step 4 Customize Your Store - Validating modules constraints loading does not show 

### Manual testing scenarios (*)
1. Open the Magento web install
2. On step 2 a message will inform that the database connection is being verified
3. On step 3 after change the admin path value, go to step 4
4. Move back to step 3 or reload the page - (the admin path value will be remain the same) 
5. Goto step 4 you will notice the message informing the verification of modules
6. Goto step 5 you will be informed about the admin user check.
